rhizome/host/bin/verify-sshable.rb

```diff
@@ -0,0 +1,41 @@
+#!/bin/env ruby
```

Comment:
Should we do the deployment in 2 steps?

1. First commit: Deploy the rhizome script
2. Second commit: Deploy the control plane change

Reply:
We have to, right? Since the control plane will just crash and the results are total meltdown until the deploy, if it depends on verify-sshable in the process.